### DISCUSSION TITLE
How is mastered topics calculated

### DISCUSSION THREAD

**Discussion by Milo123459** [lowlighter/metrics#151] [Q&A] [answered]
The title says it all... how is it calculated? I used it and it said nothing sadly 🙃

**Comment by lowlighter** [comment, marked as answer]
It's actually just a way how [starred GitHub topics](https://github.com/topics) are displayed.
`starred` mode displays them in label-like fashion while `mastered` only displays their icons (and ignore topics which doesn't have one).

The latter is more intended to "automate" your frameworks/languages/subjects knowledge and skills, but there's actually no heavy black magic behind 🙂 

If it's empty, it just mean you haven't starred any topics yet. Note that topics are counted into GitHub stars, but metrics only displays repositories stars, so it's normal if you start seeing different numbers on your profile
